spec/scenarios/codeblock/fenced/list-continuation-yaml2.adoc

```diff
@@ -0,0 +1,37 @@
+. Identify the outdated VMIs:
++
+[,bash]
+----
+  $ kubectl get vmi -l kubevirt.io/outdatedLauncherImage --all-namespaces
+----
+
+. Check the `KubeVirt` custom resource (CR) to determine whether `workloadUpdateMethods` is configured in the `workloadUpdateStrategy` stanza:
++
+[,bash]
+----
+  $ kubectl get kubevirt kubevirt --all-namespaces -o yaml
+----
+
+. Check each outdated VMI to determine whether it is live-migratable:
```

Comment:
This almost works, but there is still a missing list continuation after "Example output:".

I originally formatted the code blocks without indentation in Markdown because it was the only way I could get code blocks with no indentation in Asciidoc. So if you say that the code blocks need to be indented, then the conversion should remove the unnecessary indentation in the Asciidoc output.

Reply:
It appears that kramdown requires an indentation of 3 spaces for an ordered list.